nimbus-features/qrCodeCoordinatorRefactor.yaml

```diff
@@ -15,5 +15,5 @@ features:
           enabled: false
       - channel: developer
         value:
-          enabled: false
```

Comment:
Could you enable by default and for beta as well? So the task https://github.com/mozilla-mobile/firefox-ios/issues/16490 is done too